design/README.md

```diff
@@ -0,0 +1,54 @@
+# Crossplane Design Documents
+This directory contains all documents informing Crossplane's design. Crossplane
+design documents can be divided into two two kinds:
+
+* One-pagers. A One-pager is a brief document pitching an idea for further
+  experimentation. One-pagers, as the name suggests, are usually around one
+  page long. They provide just enough context to achieve buy-in from Crossplane
+  maintainers.
+* Design documents. A design document is a longer, more detailed design. Design
+  documents should typically be preceeded by a one-pager and/or a good amount of
+  research and experimentation.
+
+In some cases a one-pager is sufficient to move forward with a design. Complex
+or controversial designs will require a more detailed design document. Please
+name your documents appropriately when committing to this directory, i.e.
+`one-pager-my-cool-design.md` or `design-doc-my-cool-design.md`.
+
+All documents committed to this directory _must_ include the following header:
+
+```markdown
+# Document Title
+* Owner: Some Person (person@example.org)
```

Comment:
include github alias for each owner?

Reply:
Good idea - I think a Github alias is better than an email address.